zhaquirks/thirdreality/motion_sensor.py

```diff
@@ -0,0 +1,39 @@
+"""Third Reality Plug devices."""
+
+from typing import Final
+
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import QuirkBuilder
+import zigpy.types as t
+from zigpy.zcl.foundation import BaseAttributeDefs, ZCLAttributeDef
+
+
+class ThirdRealityMotionCluster(CustomCluster):
+    """Third Reality's motion sensor private cluster."""
+
+    cluster_id = 0xFF01
+
+    class AttributeDefs(BaseAttributeDefs):
+        """Define the attributes of a private cluster."""
+
+        cooldown_time: Final = ZCLAttributeDef(
+            id=0x0001,
+            type=t.uint16_t,
+            is_manufacturer_specific=True,
+            name="Cooldown_Time",

```

Comment:
The `name` is already defined by the attribute name when using this definition style, we should remove it.
```suggestion
```